acme/src/acme/client.py

```diff
@@ -248,12 +253,22 @@ def poll_finalization(self, orderr: messages.OrderResource,
             response = self._post_as_get(orderr.uri)
```

Comment:
i think we should also update the docstring of poll_finalization above because this function now works for orders which have not been finalized